src/Annotation/VersionConstraints.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace JMS\Serializer\Annotation;
+
+use Composer\Semver\Semver as ComposerSemver;
```

Comment:
I would avoid using aliases for namespaces as long as it is really needed to keep code simpler and more readable.

Reply:
Thank you for the hint. This was a leftover since I used a different name for the annotation before.